src/android/xml/mediacapture_provider_paths.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
cordova plugins usually store _res_ files into `src/android/res`: so use `src/android/res/xml/mediacapture_provider_paths.xml` instead of `src/android/xml/mediacapture_provider_paths.xml`